README.md

```diff
@@ -233,3 +233,8 @@ Code change needed to SA:
 Run (all) tests:
 
     pytest --db ingres_odbc --junit-xml=all_results_junit.xml --maxfail=12000
+
+## Known Issues
+
+Apache Superset issue [27427](https://github.com/apache/superset/issues/27427)
+The Apache Superset SQL parser is able to recognize and handle row limiting clauses that use keywords **LIMIT** and **TOP** but does not handle the **FETCH FIRST** clause, which is used by some databases, including Ingres and PostgreSQL. If a **FETCH FIRST** clause is used in a SQL statement and the Superset config parameter **SQL_MAX_ROW** is set to a value > 0, the Superset parser appends a **LIMIT** clause to the SQL statement, making it syntactically invalid. This errant behavior can be seen when working with SQL statements and table metadata in Superset SQL Lab and may occur in other Superset operations as well. It is important to note that this is a problem with Apache Superset and not with the SQLAlchemy-Ingres connector.
```

Comment:
This looks run on. The content is great! See https://github.com/ActianCorp/sqlalchemy-ingres/tree/hab6-add-known-issues#known-issues for what I mean.

Bare minimum change would be paragraph breaks:
1. one after first line
2. another one before the last line.

With those changes the markdown will be more clear. The text it self will be long in plain text so could benefit from some newlines but that's not a deal-breaker.

Reply:
Thank you for the suggestions! I added a few newlines. Hopefully it looks better now!